apps/yggdrasil/lib/yggdrasil/user.ex

```diff
@@ -25,9 +27,22 @@ defmodule Yggdrasil.User do
     |> validate_length(:password, min: 4)
     |> validate_length(:password_confirmation, min: 4)
     |> validate_confirmation(:password)
+    |> default_role
```

Comment:
Temporary, I needed the role set, but if I pass it in view the change_set it will break all the current tests. 

I think going forward we will either want two different change_sets or add it after etc. Given that we have no admin users, or the ability to assign these it suffices for now.